radish/parser.py

```diff
@@ -318,7 +318,7 @@ def _parse_scenario(self, line):
             # inherit the tags from the current feature and the explicitely
             # inherited tags given to the parser. This tags are coming from precondition scenarios
             current_tags = self._current_tags + self.feature.tags + self._inherited_tags
-            scenario_in_tags = self._tag_expr.evaluate([t.name for t in current_tags])
+            scenario_in_tags = self._tag_expr.evaluate([t.name if t.arg is None else f"{t.name}({t.arg})" for t in current_tags])
```

Comment:
does this also work for a tag which looks like this -> `@tag value` can i filter that with `--tags='tag value'` ?

Reply:
I suspect that may not work well with the filtering change. I'll add some tests this afternoon and see what happens

Reply:
damn! tag expressions does not support this syntax either. i get an exception on master aswell. 

Ill make another change to tag exp

Reply:
im not sure how that would cleanly work though, with the parens, i can determine whats an argument and whats not. but with just spaces it could get tricky with multiple 'args'

Reply:
ok i took a look and it seems good enough for me right now tag values with spaces are not supported right now update the docs for that